src/main/resources/com/google/api/codegen/java/sample.snip

```diff
@@ -64,6 +62,9 @@
       @end
 
       {@class.requestBodyTypeName} {@class.requestBodyVarName} = new {@class.requestBodyTypeName}();
+      @join field : class.requestBodyFields if class.requestBodyFields
```

Comment:
Specifically, the reason `@join if` was added was because a `@join` where no elements are available to iterate over prints an empty new line, whereas an `@join if` does not.